src/v2/Apps/Settings/Routes/EditSettings/Components/useUnlinkSettingsLinkedAccount.ts

```diff
@@ -0,0 +1,19 @@
+import { graphql } from "react-relay"
+import { useMutation } from "v2/Utils/Hooks/useMutation"
+import { useUnlinkSettingsLinkedAccountMutation } from "v2/__generated__/useUnlinkSettingsLinkedAccountMutation.graphql"
+
+export const useUnlinkSettingsLinkedAccount = () => {
+  return useMutation<useUnlinkSettingsLinkedAccountMutation>({
```

Comment:
Its funny how long it takes before things that we've done a million times get properly abstracted